src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -940,15 +940,23 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>Missing or invalid path '{0}'. Please provide a path to a project, solution file, or directory.</value>
     <comment>{0} - Input path argument</comment>
   </data>
+  <data name="Error_PathIsMissingOrInvalid_AllowAssetsFile" xml:space="preserve">
+    <value>Missing or invalid path '{0}'. Please provide a path to a project, solution, or NuGet assets file, or directory.</value>
```

Comment:
I'm reconsidering this message. The .NET SDK isn't designed to work with non-SDK style projects, and I think it's going to be a very uncommon scenario, so I don't think it's worthwhile complicating the message with this.  I'm also thinking of the poor translators who need to figure out how to translate this 😆 

Any feedback?

Reply:
I agree with your reconsideration. 

Make the happy path .NET SDK specific.